do.sh

```diff
@@ -45,14 +48,46 @@ function install_deps() {
     ansible-playbook ${ovn_fmn_playbooks}/install-dependencies.yml -i ${hosts_file}
 
     echo "-- Installing local dependencies"
-    dnf install -y docker docker-distribution
-    systemctl start docker
-    systemctl start docker-distribution
+    if yum install -y docker docker-distribution
+    then
+        systemctl start docker
+        systemctl start docker-distribution
+    else
+        yum install -y podman podman-docker
+    fi
+    yum install redhat-lsb-core python3-pip python3-virtualenv python3 python3-devel python-virtualenv --skip-broken -y
+    [ -e /usr/bin/pip ] || ln -sf /usr/bin/pip3 /usr/bin/pip
 }
 
 function configure_docker() {
-    echo "-- Configuring docker local registry on all nodes"
-    ansible-playbook ${ovn_fmn_playbooks}/configure-docker-registry.yml -i ${hosts_file}
+    echo "-- Configuring podman local registry on tester nodes"
+    if which podman
+    then
+        for container_name in `podman ps | grep -v "CONTAINER ID" | awk '{print $1}'`
+        do
+            podman stop $container_name
+            podman rm $container_name
+        done
+        [ -d /var/lib/registry ] || mkdir /var/lib/registry -p
+        podman run --privileged -d --name registry -p 5000:5000 \
```

Comment:
Shouldn't this part be moved to `install_deps()` where we install and start local dependencies? Also, I don't think we need `registries = ['localhost:5000']` on the tester node, right?

Reply:
I added this in case someone use the same system as both tester node and worker node

Reply:
and this is used to configure podman, so added it here. should I remove it to install_deps?

Reply:
Yes, I think everything that's related to the "tester" node should go to `install_deps()` in the "-- Installing local dependencies" section.